metricbeat/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '2.3'
```

Comment:
Change done in https://github.com/elastic/beats/pull/10868, kept also here to run CI.

Reply:
Awesome @jsoriano :+1: Giving you an ok already so you don't have to wait but you'll have to rebase after this https://github.com/elastic/beats/pull/10868